Makefile

```diff
@@ -329,6 +329,12 @@ ci-check: clean-downloadcache deps fast-babel check
 bumpversion: deps
 	bin/bumpversion $(VPART)
 
+CURRENT_VERSION = $(shell sed -n -e '/current_version =/ s/.*\= *// p' .bumpversion.cfg)
+CURRENT_COMMIT = $(shell git rev-parse HEAD)
```

Comment:
While these are only used here right now I could see them being useful in the near future in future additions, could you move them to the other definitions please.